pitch.md

```diff
@@ -23,38 +51,192 @@ should be more specific than "The user describes properties of unicorns.", e.g.,
+define starting coordinate!”.
 
 ## Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to create 2D line animations in this language! Specifically, it should
+be fairly easy to create multiples of a specific animation, and it should be easy to create 
+animations that have a degree of symmetry. However, since the lines are limited to straight 
+lines, it could be difficult to create more organic forms or 3D-appearing forms It could also 
+be difficult to create a lot of text using the lines (although if the user defined a rule for 
+each letter, it could be a reasonable task). Finally, it should be impossible to create animations
+using something other than lines! 
 
 ## Related work
 _Are there any other DSLs in this domain? If not, describe how you know there
 aren't and conjecture why not. If so, describe them and provide links. How well 
 do they address the need? Are there any particularly admirable qualities of the
 language? Are there parts of the language you think could be improved?_
 
+As I mentioned in my last project write-up, [Processing](https://processing.org/) is a DSL that is similar to the type 
+of language I want to create. It is a language for "learning how to code within the context of 
+the visual arts", and it allows the user to create still digital art as well as animated digital 
+art. There are many libraries that extend the language, showing its extensibility and success as 
+a readable and writable language. I actually think this DSL addresses the desire to create art and 
+animation very effectively, and I wish I had known more about this before! I suppose a drawback of 
+the language is that it has such a wide range of features that it might require some investment of 
+time to become proficient with the language, and it is not necessarily the most intuitive for those 
+who have never programmed before.
+
+The [CSS3 Animations](http://www.w3schools.com/css/css3_animations.asp) library “allows animations of most HTML elements.” It is less expressive than Processing in the sense that it can only be used to create really basic animations or animate existing elements. I think a drawback of this is that it also requires knowledge of HTML and may be intimidating for people without programming experience. However, the syntax is pretty readable and I could see how I can use elements of it in my own DSL.
+
+Finally, there is a feature in Powerpoint that allows for [line animations](https://www.screenr.com/6Rg), and if we’re playing 
+the “Is this a DSL?” game, I definitely think this would count as a DSL. Animations in powerpoint 
+are really simple to use, and they allow people without programming knowledge to add animations in
+their presentations. This is an example of a 2D line animation that was created in Powerpoint!
```

Comment:
This is really cool! I think your audience would most match Powerpoint's audience in that they don't have much programming experience but can figure out how to do things on a computer.